.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "third_party/saml2aws"]
 	path = third_party/saml2aws
-	url = https://github.com/richardcase/saml2aws.git
```

Comment:
Currently we are using a forked version of saml2aws to support Ping with NTLM:
https://github.com/richardcase/saml2aws/commit/e3668c0a9bab950e245ae8cea9599ad182da5442

We will need to either get this merged in with upstream repo or update the forked version.

Reply:
Fair enough. Is there a PR against saml2aws to get your NTLM patch in the upstream repo? Is there a reason you are maintaining the fork?

Reply:
@richardcase can you help with this query please?

Reply:
No reason for us to maintain the fork other than not having made the PR upstream.  I will try and get the rebase/PR done Monday so that we can ultimately get rid of the fork.

Reply:
hi @richardcase, any luck with submitting that PR to upstream?

Reply:
To be honest, i completely forgot about this. You may want to fork into fidelity and then create the PR from there?

Reply:
no worries, ok will do. thanks for the update. 😄

Reply:
Hopefully this will be merged some day: https://github.com/Versent/saml2aws/pull/794

Reply:
:crossed_fingers: